src/mpid/ch4/netmod/ofi/ofi_send.h

```diff
@@ -364,17 +369,19 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_send(const void *buf, MPI_Aint count, MPI
     MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_OFI_SEND);
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_OFI_SEND);
 
+    uint64_t cq_data = comm->rank;
+    MPIDI_OFI_idata_set_error_bits(&cq_data, err_flag);
```

Comment:
One observation: now the non-collective sends will also have this overhead. But I'm not fully sure about the consequences. @wesbland Can you comment on this?

Reply:
Normal send sends cq_data as well, it is just the src_rank. The only overhead is bitwise or with the err_flag. I hope (believe) the compiler should optimize that away when it is a constant 0).

Reply:
I see now. At a quick glance, I was concerned about how we might be manipulating bits and that it might have a correctness issue, but this looks fine to me.